settings.py

```diff
@@ -0,0 +1,54 @@
+import os
+from typing import Optional, Union
+
+from omegaconf import DictConfig, OmegaConf
+
+from cli import load_config
+
+
+class UFMSettings:
+    """
+    Some global run settings
+    """
+    def __init__(self):
+        """
+        config may be the loaded object or a path for loading
+        """
+        # Names and paths
+        self.ROOT_PATH = os.getcwd()
+        self.BASE_NAME = None
+        self.RUN_NAME = None
+        # Identifying file path to store info about this run
+        # Automatically determined by BASE_NAME and RUN_NAME
+        # Should not be a full path
+        self.STORE_PATH = None
+
+        self.DRY_RUN = False
+        self.SEED = None
+        self.VERBOSE = False
+        self.NON_INTERACTIVE = False
+
+        return
```

Comment:
I think we may just not need this class/file. How I tend to prefer to do this kind of thing is with a bunch of parameters defined with argparse, then load them with wandb.init()--see https://docs.wandb.ai/guides/track/config#set-the-configuration-with-argparse. Then, Weights & Biases will handle all the config saving and overriding for us.

Reply:
Wouldn't this require passing the config object to basically every function if they want to query any settings/configs? I think Rohan has already set up config parsing with OmegaConf

Reply:
`wandb.config` is a dict available globally, so we don't even have to pass it in. See https://docs.wandb.ai/guides/track/config

Reply:
ok added